BTCPayServer.Client/Models/CreateInvoiceRequest.cs

```diff
@@ -31,9 +31,9 @@ public class CheckoutOptions
             public TimeSpan? Monitoring { get; set; }
 
             public double? PaymentTolerance { get; set; }
-
             [JsonProperty("redirectURL")]
             public string RedirectURL { get; set; }
+            public string DefaultLang { get; set; }
```

Comment:
can we name this better now?
`
```suggestion
            public string DefaultLanguage { get; set; }
```

Reply:
Actually I started doing that, but "Lang" is commonly used. No strong pref though.